examples/raft/tests/custom_test_tensorleap.py

```diff
@@ -0,0 +1,60 @@
+import matplotlib.pyplot as plt
+from tensorleap import *
+import onnxruntime as rt
+from os import environ
+from os.path import exists
+import urllib
+
+if __name__ == "__main__":
+    # This test requires the relevant secret to be loaded to the system environment AUTH_SECRET
+    if environ.get('AUTH_SECRET') is None:
+        print("The AUTH_SECRET system variable must be initialized with the relevant secret to run this test")
+        exit(-1)
+    data_subsets: List[PreprocessResponse] = subset_images()  # preprocess and get data preprocess response list
+    i, scene_flow = 0, data_subsets[0]
+    x = subset_images()
+    # get inputs
+    img_1 = np.expand_dims(input_image1(i, scene_flow), axis=0)
+    img_2 = np.expand_dims(input_image2(i, scene_flow), axis=0)
+    # run inference on Onnx
+    if not exists("raft.onnx"):
+        print("Downloading Raft ONNX for inference")
+        urllib.request.urlretrieve(
+            "https://storage.googleapis.com/example-datasets-47ml982d/raft/raft_new.onnx",
+            "raft.onnx")
```

Comment:
Is this a public bucket that anyone can reach?

Reply:
Yes